openapi.yaml

```
@@ -1617,7 +1617,7 @@ paths:
                                             example: "test.txt"
                                         attachment:
                                             type: string
-                                            description: base64 encoded file contents
+                                            description: base64 encoded file contents with 30 MB limit
```

This may be the most absurd nit in history so I apologize...

Can we capitalize the "b" in base64 🤩😅?
```suggestion
                                            description: Base64 encoded file contents with 30 MB limit
```